README.rst

```diff
@@ -0,0 +1,77 @@
+===========
+itemloaders
+===========
+
+.. image:: https://img.shields.io/pypi/v/itemloaders.svg
+   :target: https://pypi.python.org/pypi/itemloaders
+   :alt: PyPI Version
+
+.. image:: https://img.shields.io/pypi/pyversions/itemloaders.svg
+   :target: https://pypi.python.org/pypi/itemloaders
+   :alt: Supported Python Versions
+
+.. image:: https://travis-ci.com/scrapy/itemloaders.svg?branch=master
+   :target: https://travis-ci.com/scrapy/itemloaders
+   :alt: Build Status
+
+.. image:: https://codecov.io/github/scrapy/itemloaders/coverage.svg?branch=master
+   :target: https://codecov.io/gh/scrapy/itemloaders
+   :alt: Coverage report
+
+.. image:: https://readthedocs.org/projects/itemloaders/badge/?version=latest
+   :target: https://itemloaders.readthedocs.io/en/latest/?badge=latest
+   :alt: Documentation Status
+
+
+``itemloaders`` is a library that helps you collect data into models.
+
+It's specially useful when you need to standardize the data from many sources.
+For example, it allows you to have all your casting and parsing rules in a
+single place.
+
+Also, it comes in handy to extract data from web pages, as it supports
+data extraction using CSS and XPath Selectors.
```

Comment:
This description is a bit confusing to me. Are we suggesting that itemloaders is a general thing, not related to web scraping, which may also come handy for web scraping?

Reply:
I guess it is not restricted to web scraping.
If I want to load a `dict` from a XML source, it could be used, right?
Similarly to read from a JSON source or something else..

So, we can have the description related to web scraping or leave it open as a library to standardize the process of extracting/loading data from a source

Reply:
What about explicitly mentioning HTML and XML as the sources of data in the first paragraph, and in the third paragraph replace “comes in handy” with “is specially useful” and move the CSS and XPath part to the first paragraph?

Reply:
Updated @Gallaecio